packages/client/src/comp/WarningMsg.tsx

```diff
@@ -87,12 +90,31 @@ const RawWarningMsg: React.FC<RawWarningProps> = ({ setOpen }) => {
 export const WarningMsg = () => {
   const { initialData } = InitialDataContainer.useContainer()
   const { path } = useAppHistory()
-  const [ open, setOpen ] = React.useState(false)
+  // Since localStorage doesn't trigger re-renders, users visiting the
```

Comment:
Can we pull this logic into an Unstated that just exposes

```
open
toggleOpen
```

and handles visited and localStorage internally without exposing it?  Also, let's write a unit test since this is the kind of thing that type systems don't check for us.